web/views.py

```
@@ -6488,3 +6490,34 @@ def delete_post(request, post_id):
     if request.method == "POST":
         post.delete()
     return redirect("social_media_dashboard")
+
+
+@require_GET
+def get_counter_data(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_counter_data`

(ANN201)